src/audio/copier/copier.c

```diff
@@ -1408,6 +1413,24 @@ static void copier_dma_cb(void *arg, enum notify_id type, void *data)
 	/* callback for one shot copy */
 	if (cd->hd->copy_type == COMP_COPY_ONE_SHOT)
 		host_one_shot_cb(cd->hd, bytes);
+
+	/* apply attenuation since copier copy missed this with host device remove */
```

Comment:
is this a fix for an earlier commit in this PR or for an existing bug? Is it related to this commit?

Reply:
not early fix, not existing bug, just keep SOF functionality, in this change, if you see copier_copy, attenuation will be missed, so added it here to keep attenuation with this change.